code/modules/mob/dead/observer/observer.dm

```diff
@@ -410,7 +410,11 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	can_reenter_corpse = FALSE
 	// Update med huds
 	var/mob/living/carbon/current = mind.current
-	current.med_hud_set_status()
+	if(current)
+		current.med_hud_set_status()
+		current.log_message("Had their player ([key_name(src)]) do-not-resuscitate", LOG_GAME, color = COLOR_GREEN)
+		log_message("Has opted to do-not-resciscitate from their body ([current])", LOG_GAME, color = COLOR_GREEN)
```

Comment:
```suggestion
		current.log_message("had their player ([key_name(src)]) do-not-resuscitate", LOG_GAME, color = COLOR_GREEN)
		log_message("has opted to do-not-resciscitate from their body ([current])", LOG_GAME, color = COLOR_GREEN)
```
This will probably look better based on the screenshot you posted in coderbus:

![image](https://user-images.githubusercontent.com/34697715/181388371-e8412757-6900-4c4c-86da-47f40a6e91da.png)